src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.search-big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.search-big__input {
+  width: 100%;
+  height: 70px;
+  padding-left: 63px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+.search-big__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-big__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-big input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.search-big__btn {
+  position: absolute;
+  top: 0;
+  left: 1px;
+  border: none;
+  cursor: pointer;
+  padding: 25px;
+}
+
+.search-small {
+  position: relative;
+}
+
+.search-small svg {
+  width: 11px;
+  height: 11px;
+  border: none;
+}
+
+.search-small__input {
+  height: 42px;
+  width: 100%;
+  padding-left: 33px;
+  padding-bottom: 2px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-small__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-small__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-small__input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 500;
+  font-size: 14px;
+}
+
+.search-small__btn {
+  position: absolute;
+  top: 0;
+  left: 3px;
+  border: none;
+  padding: 10px;
+  cursor: pointer;
```

Comment:
check this checklist point
<img width="827" alt="image" src="https://user-images.githubusercontent.com/102411130/221788107-daf7d001-c684-44e8-a0c9-1fe76dbca93a.png">